.env.test

```diff
@@ -85,6 +85,8 @@ DEFAULT_NUMBER_OF_RECORDS=100
 EMAIL_SUBJECT_PREFIX="[Simple TEST]"
 RAILS_MAX_THREADS=20
 SECRET_KEY_BASE=b90c991f1987f5be558af41e1fdc73741adbef2285f30a5bd500dcd7804b2ef1b49e8b07e03bd7154dcecd5050102f0f59cd182c8b40e48a66d416228dbd33fe
+USER_AUTH_LOCKOUT_IN_MINUTES=20
+USER_AUTH_MAX_FAILED_ATTEMPTS=5
```

Comment:
If you go with the ["optional configuration" approach](https://github.com/simpledotorg/simple-server/pull/975/files#r428483384), we can remove these two from here.